src/main/java/com/knu/buga1chuk/data_structure/LinkedList.java

```diff
@@ -115,7 +121,8 @@ public int getLinkedListLength() {
     public void print() {
         Node currentNode = head;
         while (currentNode != null) {
-            System.out.print(currentNode.data + " ");
+            LOG.info((currentNode.data + " "));
```

Comment:
Log in loop is a bad practice.
Refactor it with the `StringBuilder`.

Reply:
fix